consensus/polybft/proto/messages.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
```

Comment:
Please coordinate with `edge` team to make sure we are using the same protobuf version since you have updated all the protobuf objects of the repo.

Reply:
No specific version is used, any latest version is fine.